app/Http/Middleware/EnsureManager.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class EnsureManager
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure(\Illuminate\Http\Request): (\Illuminate\Http\Response|\Illuminate\Http\RedirectResponse)  $next
+     * @return \Illuminate\Http\Response|\Illuminate\Http\RedirectResponse
+     */
+    public function handle(Request $request, Closure $next)
+    {
+        // パラメータの`{team}`を取得（←学習のため）
+        $team = $request->route()->parameter('team');
```

Comment:
```suggestion
        $team = $request->route('team');
```

こちらでも同様になるはずなので試してみてください（短縮記法のようです）